src/main/java/core/basesyntax/io/CsvReader.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.io;
+
+import core.basesyntax.io.exception.CsvException;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.OperationType;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CsvReader {
+    private static final String DELIMITER = ",";
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+
+    public List<Operation> readOperations(String fileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
```

Comment:
It's recommended to use try-with-resources with classes that implement the `AutoCloseable` interface, but `FileReader` should be also included in try-with-resources to ensure that the file is closed properly in case an exception occurs. Modify the code to include `FileReader` in the try-with-resources statement.